### ISSUE TITLE
Support BGR48 output format

### ISSUE THREAD

**Issue by CrendKing** [Nevcairiel/LAVFilters#391]
The "[RGB32](https://github.com/Nevcairiel/LAVFilters/blob/88da52ae317c335929dadc655dda92a709e6b7e8/decoder/LAVVideo/pixconv/convert_generic.cpp#L56)" and "[RGB24](https://github.com/Nevcairiel/LAVFilters/blob/88da52ae317c335929dadc655dda92a709e6b7e8/decoder/LAVVideo/pixconv/convert_generic.cpp#L60)" from LAV filters are both B-G-R ordering, while the "RGB48" in LAV filters is actually [AV_PIX_FMT_RGB48LE](https://github.com/Nevcairiel/LAVFilters/blob/88da52ae317c335929dadc655dda92a709e6b7e8/decoder/LAVVideo/pixconv/convert_generic.cpp#L74), which in ffmpeg is defined as "[packed RGB 16:16:16, 48bpp, 16R, 16G, 16B, the 2-byte value for each R/G/B component is stored as little-endian](https://github.com/FFmpeg/FFmpeg/blob/6dad42854690f5754a1cf1ed138ebd9813e1af37/libavutil/pixfmt.h#L103)". 

The R-G-B ordering has some compatibility issue with other library. For example, [AviSynth+ only supports B-G-R](http://avisynth.nl/index.php/Avisynthplus_color_formats). So to feed AviSynth+ the output of LAV filters would requires a costly pixel-by-pixel conversion.

Is there any chance LAV filters can add a "BGR48" format that converts output to AV_PIX_FMT_BGR48LE?

**Comment by Nevcairiel** [comment]
I do not plan to add additional output formats. As you noted conversions are costly, and LAVs primary goal is to output the native format that matches the encoded video. Converting to the RGB48 format is already rather inefficient since its not a hand-optimized case.

The format was mostly added to fill the gap of a 16-bit RGB format, and was the logical (eg. fastest) option when converting from xyz12 DCP content to RGB using ffmpeg.

TL;DR:
Do not rely on LAV to convert. Accept the native format and do the conversion as you require them.
